store/driver/sql_fail_test.go

```diff
@@ -100,7 +100,6 @@ func (s *testSQLSerialSuite) TestFailBusyServerCop(c *C) {
 }
 
 func TestMain(m *testing.M) {
-	tikv.ReadTimeoutMedium = 2 * time.Second
```

Comment:
why remove this?

Reply:
`ReadTimeoutMedium` should be a `const`, but with this line it must be a var. I find it is useless to change this value in this test, so I remove it.

Reply:
Looks it can make tests run longer.

Reply:
I‘ve tried the related test cases on my computer, it runs even faster than before.